src/functions-and-arrays.js

```diff
@@ -52,16 +93,26 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(wordsUnique) {
+  let arr=[]
+for (let i=0; i<wordsUnique.length;i++){
+    if(wordsUnique.indexOf([i]!== -1)) {
+      return arr.push([i])
+    }
+  }
+}
```

Comment:
- Aquí no te funciona, porque cuando comparas el `indexOf()` con -1, indica que esa palabra está en el array, pero una palabra que está duplicada siempre te va a dar un valor diferente a -1, puesto que **sí** está en el array.
- La clave la tienes en que el `indexOf` siempre te devuelve la primera ocurrencia del elemento, por lo que si un elemento se repite varias veces, solo te devolvera la primera posición en la que ese elemento aparezca, que debería ser la misma que la de la `i` del bucle for que has definido, así que ahí tienes la clave ya que si los comparas y no son iguales, significará que ese elemento no es la primera vez que aparece!